controllers/configurationpolicy_controller.go

```diff
@@ -783,41 +748,69 @@ func (r *ConfigurationPolicyReconciler) handleObjectTemplates(plc policyv1.Confi
 
 		policyUserErrorsCounter.WithLabelValues(parent, plc.GetName(), "invalid-template").Add(1)
 
-		return
+		return fmt.Errorf("%w: %s", ErrPolicyInvalid, validationErr)
 	}
 
-	// object handling for when configurationPolicy is deleted
-	if plc.Spec.PruneObjectBehavior == "DeleteIfCreated" || plc.Spec.PruneObjectBehavior == "DeleteAll" {
-		var crdDeleting bool
+	//nolint: wastedassign
```

Comment:
Does using `var usingWatch bool` resolve the linter?

Reply:
Times like this I wish go had a ternary operator

Reply:
Setting this to `var usingWatch bool` seems to work. I tend to initialize to false based on previous PR review feedback as newcomers to Go don't know that `var usingWatch bool` is equivalent to a false boolean.

Reply:
Right, and I agree that's a good practice, but I think it's complaining because there's a conditional block where no matter what happens, it's getting assigned, so it doesn't matter what it's initialized to.

Reply:
Yeah the `var` syntax can be annoying... I just dislike `//nolint` even more (when avoidable).

Reply:
Yeah the `var` syntax can be annoying... I just dislike `//nolint` even more (when avoidable).